libs/vertexai/pyproject.toml

```diff
@@ -17,6 +17,11 @@ google-cloud-aiplatform = "^1.47.0"
 google-cloud-storage = "^2.14.0"
 # optional dependencies
 anthropic = { extras = ["vertexai"], version = ">=0.23.0,<1", optional = true }
+pandas = { version = ">= 1.0.0", optional = true }
+google-cloud-bigquery-storage = { version = ">=2.6.0,<3",  optional = true }
+pyarrow = { version = ">= 6.0.1", optional = true }
+langchain-community = { version = ">=0.0.28", optional = true }
```

Comment:
we shouldn't depend on community

Reply:
Without that, we would need to bring this function our side though 

https://github.com/langchain-ai/langchain/blob/37cfc003107ea800953be912f2eebfbf069c9587/libs/community/langchain_community/vectorstores/utils.py#L23

Wdyt?

Reply:
took out MMR search for the time being